runtime/src/main/java/io/quarkiverse/renarde/util/AuthenticationFailedExceptionMapper.java

```diff
@@ -55,15 +56,17 @@ private Response redirectToRoot(String message) {
         flash.flash("message", message);
         // FIXME: URI, perhaps redirect to login page?
         ResponseBuilder builder = Response.seeOther(URI.create("/"));
-        // FIXME: constant
-        NewCookie logoutCookie = new NewCookie("QuarkusUser", "", "/", null, NewCookie.DEFAULT_VERSION, null,
-                NewCookie.DEFAULT_MAX_AGE, null, false, false);
-        builder.cookie(logoutCookie);
+        builder.cookie(createLogoutCookie("QuarkusUser"));
         Map<String, Object> map = new HashMap<>();
         // FIXME: format?
         map.put("message", message);
         Flash.setFlashCookie(request, request.response(), map);
 
         return builder.build();
     }
+
+    public static NewCookie createLogoutCookie(String cookieName) {
```

Comment:
```suggestion
    public static NewCookie invalidateCookie(String cookieName) {
```

Perhaps that's a better name?

Reply:
Yeap, that's a better one.